fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java

```diff
@@ -941,4 +943,17 @@ public DataPartition constructInputPartitionByDistributionInfo() throws UserExce
             return DataPartition.RANDOM;
         }
     }
+
+    @Override
+    public void initOutputSlotIds(Set<SlotId> requiredSlotIdSet, Analyzer analyzer) throws NotImplementedException {
```

Comment:
Although fe calculates the output slot of the scan node, if be does not make any changes, the function of project cannot be realized.

Reply:
OK, I'll add supports in BE later